quarkchain/cluster/rpc.py

```diff
@@ -76,13 +76,15 @@ class SlaveInfo(Serializable):
         ("host", PrependedSizeBytesSerializer(4)),
         ("port", uint16),
         ("chain_mask_list", PrependedSizeListSerializer(4, ChainMask)),
+        ("type", PrependedSizeBytesSerializer(4)),
```

Comment:
we probably don't want to have the type information for now. reasons is `SlaveInfo` RPC is used to establish full connections between master/slaves, however at our first step we don't have full mechanism to do this with libra slave

so I suggest when establishing connections, skip slaves if their type is not the default.

Reply:
if I skip connection them, then __has_all_shards() checks failed, the cluster will be shut down.

Reply:
then you should have "normal" slaves covering all shards, but have separate additional slaves to make grpc calls

Reply:
done